servo-build-dependencies/android.sls

```diff
@@ -28,6 +28,7 @@ android-dependencies:
       - lib32z1
       - libstdc++6
       - libgl1-mesa-dev
+      - unzip
```

Comment:
To be more clear, please use the Salt `archive.extracted` state (with `archive_type: zip`) instead of manually downloading and extracting for `android-ndk`. (`android-sdk` has an example of this in the same file, and the docs are [here](https://docs.saltstack.com/en/2016.3/ref/states/all/salt.states.archive.html#salt.states.archive.extracted).) Also keep the `unzip` package dependency, as Salt sometimes uses it internally.